config/initializers/services.rb

```diff
@@ -22,3 +22,6 @@ class ServiceNotRegisteredException < Exception; end
 
 require 'gds_api/publishing_api'
 CollectionsPublisher.services(:publishing_api, GdsApi::PublishingApi.new(Plek.new.find('publishing-api')))
+
+require 'gds_api/panopticon'
+CollectionsPublisher.services(:panopticon, GdsApi::Panopticon.new(Plek.new.find('panopticon'), bearer_token: 'example'))
```

Comment:
We'll need some mechanism to specify the real bearer token at deploy time, ideally without overwriting this whole file.

As an aside, the `services` method definition above really doesn't belong in here, but that's unrelated to this PR.